_rules/button-accessible-name-97a4e1.md

```diff
@@ -117,41 +117,41 @@ Off screen elements should be tested.
 
 #### Passed Example 8
 
-Button has [accessible name][] that is not empty (`""`).
+This `input` element has an [accessible name][] because of the default accessible name for an `input` element with a `type` attribute set to `reset`.
```

Comment:
This is the case where the accessible name does not come solely from the [Accessible name Computation algorithm](https://www.w3.org/TR/accname-aam-1.1/) but also from the [HTML AAM](https://www.w3.org/TR/html-aam-1.0/#input-type-button-input-type-submit-and-input-type-reset).

Our current definition of accessible name only refers to the algorithm, not to the AAM. Thus, per our current definition, this has an empty "accessible name (ACT-R version)"…

=> We need to update the definition to also refer to the AAM.
This is likely the topic for another PR, however (#1096)…